.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 repos:
 # Official repo for the clang-format hook
 - repo: https://github.com/pre-commit/mirrors-clang-format
-  rev: 'v8.0.1'
+  rev: 'v19.1.2'
```

Comment:
This upgrade would mean reformatting the codebase, which is not triggered here because no C++ files are changed in this PR.

What is the motivation for updating clang-format?

Reply:
Unintended change from something I was trying out on a different branch. I will revert this.

Reply:
I do not disagree with this, I just think we should do it in another PR to get this merged first.

Reply:
Okay, we can do it in another PR. It would also make sense to merge https://github.com/precice/calculix-adapter/pull/118 before updating clang-format.